krm-functions/lib/nad/v1/nad_test.go

```diff
@@ -0,0 +1,121 @@
+/*
+Copyright 2023 Nephio.
+Copyright 2023 Samsung.
```

Comment:
You should check with your internal team if this is needed, or if "Nephio Project Authors" is sufficient. At least Google does not require this for our contributions; apparently these headers are actually not particularly meaningful for legal reasons (that is not legal advice...). The administrative overhead of carrying a lot of these notices from every contributor is quite painful; thus, changing it all to a generic "project authors" and using the Git history is preferred. For example, see: https://github.com/GoogleContainerTools/kpt/pull/3907

Now, if we are pulling this file from something that someone at Samsung wrote, and they have this header, and it is not someone at Samsung doing the commit, then we have to keep the header, according to the Apache license. That is, only someone from Samsung could remove this.

Sicne Wim is doing the commit, we may need it. But If Ganesh is doing the commit then it wouldn't be needed - assuming Wim copied this from something Ganesh wrote somewhere else.

Reply:
Hi John, I too think having "Nephio Project Authors" is sufficient. I will also check this internally with my organisation. 
This practise came from contribution towards ONAP, where anyone updating a file will append their organisation’s Copyright on top. Please note in ONAP, there is no concept of PR, it was all direct review and commit. But in Nephio, a PR can have many commits from multiple authors. In summary, I agree with your legal reasoning and happy to remove the same from this file. We will just kept Nephio’s Copyright on top. Thanks.